src/app/organizer/Columns.tsx

```diff
@@ -574,11 +576,19 @@ export function getColumns(
         const inloadouts = loadoutsByItem[item.id];
         return (
           inloadouts &&
-          inloadouts.length > 0 &&
-          inloadouts.map((l) => <div key={l.loadout.id}>{l.loadout.name}</div>)
+          inloadouts.length > 0 && (
+            <LoadoutsCell loadouts={inloadouts.map(({ loadout }) => loadout)} owner={item.owner} />
```

Comment:
```suggestion
            <LoadoutsCell loadouts={inloadouts.map((l) => l.loadout)} owner={item.owner} />
```
think this is shorter and 2% less confusing